File: 485-max-consecutive-ones/max-consecutive-ones.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) 
    {
        int ans = 0;
        int maxx = INT_MIN;
        for(int i = 0; i < nums.size(); i++){
            if(nums[i] == 1){
                ans += 1;
            }
            else{
                maxx = max(maxx, ans);
                ans = 0;
            }
        }
        maxx = max(maxx, ans);
        return maxx;
    }
};

File: 485-max-consecutive-ones/max-consecutive-ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int n = nums.length;
        int cnt = 0;
        int maxx = 0;
        for(int i = 0; i < n; i++) {
            if(nums[i] == 1){
                cnt += 1;
            }
            else{
                cnt = 0;
            }
            maxx = Math.max(maxx, cnt);
        }
        return maxx;
    }
}